### PR TITLE
Reinis Kalvins News site

### PR DESCRIPTION
**Pull request by Meeteyes**
I tried to experiment with basics from all the concepts covered in this weeks class.

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -2,21 +2,93 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>Binoklis</title>
+    <link rel="shortcut icon" type="image/png" href="./binoklis-logo.png"/>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
```
Comment (lenisili) [inline code/index.html:8]: I like the fonts you have chosen; clear & nice

File: code/index.html [hunk 2]
```diff
@@ -2,21 +2,93 @@
...
+      </div>
+      <div class="big-news-text">
+        <h1>JRT play avalible in VR</h1>
+        <p>The performance LATVIAN LOVE is based on true events and actors’ own fantasies. Materials obtained during interviews and meetings – personal stories, letters, photographs – served as a source for the actors’ fantasy and work on the parts. The performance consists of 13 scenes, each presenting different dating situation and social group. All the characters represent some true to type personalities one can find somewhere next to us either in Riga or in country-side.</p>
+      </div>
     </section>
 
-    <section class="">
-      <!-- The other news  -->
+    <section class="small-news">
```
Comment (lenisili) [inline code/index.html:39]: The structure is very clear and class names are well defined

File: code/index.html [hunk 3]
```diff
@@ -2,21 +2,93 @@
...
+        <h2>Headline</h2>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas semper metus quis felis egestas scelerisque. Aenean consequat odio a ipsum feugiat suscipit. Phasellus odio ex, dignissim id pharetra vitae, imperdiet tincidunt augue. Donec ligula arcu, ullamcorper nec bibendum sed, pulvinar nec lorem. In tristique arcu.</p>
+      </div>
+      <div class="small-news-item">
+        <img class="small-news-picture" src="https://images.unsplash.com/photo-1538600375473-cdc68cf8bf81?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1050&q=80" alt="documentary film screening" />
+        <h2>Headline</h2>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas semper metus quis felis egestas scelerisque. Aenean consequat odio a ipsum feugiat suscipit. Phasellus odio ex, dignissim id pharetra vitae, imperdiet tincidunt augue. Donec ligula arcu, ullamcorper nec bibendum sed, pulvinar nec lorem. In tristique arcu.</p>
+      </div>
     </section>
+    <footer class="bottom-footer">
```
Comment (lenisili) [inline code/index.html:81]: I really liked your idea too add social media logos in the footer

File: code/index.html [hunk 4]
```diff
@@ -2,21 +2,93 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Carter+One&family=Roboto:wght@300&display=swap" rel="stylesheet">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
   </head>
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+    <header class="header">
+      <img class="logo-image" src="./binoklis-logo.png" alt="Logo of the company- binoculars" />
```
Comment (lenisili) [inline code/index.html:19]: Where does the name binoculars come from? I was a bit confused because I thought the name of the company s binoklis

File: code/style.css [hunk 5]
```diff
@@ -1,4 +1,222 @@
...
+  padding: 10px 60px 10px 20px;
+  font-family: 'Carter One', cursive;
+  color: #F2002A;
+}
+
+.logo-image{
+  height: 100%;
+}
+
+nav{
```
Comment (lenisili) [inline code/style.css:53]: I think it is great idea that the links are not displayed in the mobile view. Why did you not add a burger menu instead?

File: code/style.css [hunk 6]
```diff
@@ -1,4 +1,222 @@
...
+}
+
+p{
+  text-align: justify;
+  /* padding: 10px 0 0 0; */
+}
+ 
+/* following formating is affecting the big news section */
+.big-news{
+  padding: 40px 10px 40px 10px;
```
Comment (lenisili) [inline code/style.css:68]: I think the code would work also if you just write
`padding: 40px 10px`

File: code/style.css [hunk 7]
```diff
@@ -1,4 +1,222 @@
...
+
+.big-news-visual{
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.big-news-picture{
+  width: 95%;
+  -webkit-transform: scale(1);
```
Comment (lenisili) [inline code/style.css:84]: very nice!

File: code/style.css [hunk 8]
```diff
@@ -1,4 +1,222 @@
...
+	transform: scale(1);
+	-webkit-transition: .3s ease-in-out;
+	transition: .3s ease-in-out;
+  box-shadow: 0 0 0 8px white;
```
Comment (lenisili) [inline code/style.css:88]: good idea to add a shadow

File: code/style.css [hunk 9]
```diff
@@ -1,4 +1,222 @@
...
+  border-radius: 3px;
+}
+
+.big-news-picture:hover{
```
Comment (lenisili) [inline code/style.css:92]: I very much like the hover effect

File: code/style.css [hunk 10]
```diff
@@ -1,4 +1,222 @@
...
+}
+.big-news-text{
+  /* width: 95%; */
+  padding: 8px;
+}
+
+/* following formating is affecting the small news sections */
+.small-news{
+  display: grid;
+  grid-template-columns: 1fr;
```
Comment (lenisili) [inline code/style.css:104]: nice to also add some grid elements

File: code/style.css [hunk 11]
```diff
@@ -1,4 +1,222 @@
...
+
+.footer-image{
+  width: 30px;
+  height: 30px;
+}
+
+.small-logo{
+  width: 100%;
+}
+/* ------i-PAD VERSION ------------*/
```
Comment (lenisili) [inline code/style.css:161]: Why do you call iPad version instead of Tablet version? In general I think your description of the code is great

File: code/style.css [hunk 12]
```diff
@@ -1,4 +1,222 @@
...
+@media (min-width: 768px){
+  /* all the changes that are affecting the header container and child elements */
+  .header{
+    justify-content: space-around;
+    height: 80px;
```
Comment (lenisili) [inline code/style.css:166]: Why did you made the header smaller with the tablet version?

File: code/style.css [hunk 13]
```diff
@@ -1,4 +1,222 @@
...
+  }
+  .logo-image{
+    display: inline-block;
```
Comment (lenisili) [inline code/style.css:169]: I really like the way how you put the logo in the tablet version!

File: code/style.css [hunk 14]
```diff
@@ -1,4 +1,222 @@
...
+    height: 100%;
+  }
+
+  nav{
+    display: flex;
+    justify-content: space-around;
+    width: 100%;
+    position: relative;
```
Comment (lenisili) [inline code/style.css:177]: It is a great idea to ad the position attribute.

File: code/style.css [hunk 15]
```diff
@@ -1,4 +1,222 @@
+*{
+  margin: 0;
+  padding: 0;
```
Comment (lenisili) [inline code/style.css:3]: Why did you add a 0 padding?

File: code/style.css [hunk 16]
```diff
@@ -1,4 +1,222 @@
...
+  font-size: 30px;
+}
+.header{
+  width: 100%;
+  background-color: white;
+  height: 100px;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: flex-start;
```
Comment (lenisili) [inline code/style.css:43]: Isn't that the default value in regard to the justify-content?

File: code/style.css [hunk 17]
```diff
@@ -1,4 +1,222 @@
...
+.small-news-item{
+  padding: 15px;
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+  font-size: 12px;
+  justify-content: center;
+}
+
+.small-news-picture-container{
```
Comment (lenisili) [inline code/style.css:118]: Why did you define the container? As it is seems it does not make a difference to the design?

File: code/style.css [hunk 18]
```diff
@@ -1,4 +1,222 @@
...
+}
+
+.gap-line{
+  width: 100%;
+  height: 5px;
+  margin: 0 20% 0 20%;
+  background-color: white;
+}
+
+.bottom-footer{
```
Comment (lenisili) [inline code/style.css:145]: I really like the design of the footer

File: code/style.css [hunk 19]
```diff
@@ -1,4 +1,222 @@
...
+    width: 50%;
+  }
+
+  /* all the changes that affect the small-news contianer CSS GRID used */
+  .small-news{
+    grid-template-columns: 1fr 1fr;
+  }
+
 }
+/* ---------DESKTOP VERSION ---------*/
```
Comment (lenisili) [inline code/style.css:203]: The changes to the different media queries are very precise and well defined. It is all there what's needed but not more than necessary. Great job!

**Review by lenisili — COMMENTED** [review]
You did a great job, the page looks really fine. I especially liked how you combined the flex and the grid approach and how you added changes to the different media queries. You definitely fulfilled all the requirements and the code was from the first moment on very understandable.

File: code/style.css (hunk 5 above)
Comment (Meeteyes) [reply code/style.css:53]: In this weeks website, I will add a burger, I was a bit lazy :)

File: code/style.css (hunk 15 above)
Comment (Meeteyes) [reply code/style.css:3]: This way I override the default margin and padding formatting that the elements in my HTML might have. I read it is a good practice to do in order to avoid some mysterious default margins or padding popping up on some elements.

File: code/style.css (hunk 16 above)
Comment (Meeteyes) [reply code/style.css:43]: flex-start is default, yes.

File: code/style.css (hunk 17 above)
Comment (Meeteyes) [reply code/style.css:118]: I haven't cleaned it up. I had a container defined in HTML, but then I removed it. This is not used on the site.
